components/datetime/src/date.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(Default)]
+pub struct DateTime {
```

Comment:
An interesting question is going to be how we represent the rest of the fields: weekday, all three kinds of day period, era, and so forth.

Reply:
Yes. I don't know about that at all, and would love your help in https://github.com/unicode-org/icu4x/issues/240